test/call.c

```diff
@@ -365,6 +366,17 @@ static void event_handler(struct ua *ua, enum ua_event ev,
 
 		break;
 
+	case UA_EVENT_CALL_REMOTE_SDP:
+		if (f->accept_session_updates &&
+		    call_state(call) == CALL_STATE_ESTABLISHED &&
+		    !call_is_outgoing(call) &&
+		    sdp_media_ldir(stream_sdpmedia(video_strm(call_video(
+				call)))) == SDP_SENDRECV &&
+		    sdp_media_rdir(stream_sdpmedia(video_strm(call_video(
+				call)))) == SDP_INACTIVE) {
```

Comment:
Maybe we could use a "state" variable set in the test-routine instead of the long if condition.

Reply:
I tried and couldn't get it to work. These events are triggered too often and not when I would expect it.